### PR TITLE
fix: Fix the regression bug of soscleaner IP obsfuscating

### PR DESCRIPTION
**Pull request by xiangce**
Signed-off-by: Xiangce Liu <xiangceliu@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [x] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

fix: #3346

### CODE REVIEW

**Comment by chenlizhong** [comment]
@xiangce  The code need a little enhancement to sort the new ips also.  For the ip ends with `.1`, It is always means gateway.  Current code will replace `.1` with new `.2 ` ip.  The result is different after replacement for human.

When input below line,
`tcp6       0      0 172.16.1.**1**:23           172.16.1.197:63564      ESTABLISHED 0          118627890  1/systemd`

The function will return below line.
`tcp6       0      0 10.230.230.**2**:23           10.230.230.1:63564      ESTABLISHED 0          118627890  1/systemd`

In fact, we expect below line
``tcp6       0      0 10.230.230.**1**:23           10.230.230.2:63564      ESTABLISHED 0          118627890  1/systemd``

**Comment by chenlizhong** [comment]
After discuss with xiangce, sorting new ips can not resolve below kind of line. So I agree keep current changes. 

`tcp6 0 0 192.0.0.1:23   10.0.0.2:63564 ESTABLISHED 0 118627890 1/systemd`

File: insights/tests/test_soscleaner.py [hunk 1]
```diff
@@ -35,6 +35,42 @@ def test_sub_ip_match(line, expected):
...
+
+@mark.parametrize(("line", "expected"), [
+    (
+        "tcp6       0      0 10.0.0.1:23           10.0.0.110:63564   ESTABLISHED 0",
+        "tcp6       0      0 10.230.230.2:23       10.230.230.1:63564 ESTABLISHED 0"
+    ),
+    (
+        "tcp6  10.0.0.11    0 10.0.0.1:23       10.0.0.111:63564    ESTABLISHED 0",
+        "tcp6  10.230.230.2 0 10.230.230.3:23   10.230.230.1:63564  ESTABLISHED 0"
+    ),
```
Comment (Glutexo) [inline insights/tests/test_soscleaner.py:66]: As you can see in the command fixtures above, I kept the lines short in the code even though the output contained long lines.

File: insights/tests/test_soscleaner.py [hunk 2]
```diff
@@ -35,6 +35,42 @@ def test_sub_ip_match(line, expected):
...
+        (
+            "        inet 10.0.2.155"
+            "  netmask 10.0.2.1"
+            "  broadcast 10.0.2.15"
+        ),
+        (
+            "        inet 10.230.230.1"
+            "  netmask 10.230.230.3"
+            "  broadcast 10.230.230.2"
+        ),
```
Comment (Glutexo) [inline insights/tests/test_soscleaner.py:49]: Now, the `ip addr` command output occurs multiple times in the tests. I’d consider extracting that to a constant or a generator function.

**Pull request merged**